apollo-router/src/executable.rs

```diff
@@ -261,8 +260,11 @@ APOLLO ROUTER v{version}
 
     $ ./router --supergraph starstuff.graphql
 
-    "#,
-            ));
+    "#
+            );
+            // The reason to exit here is that anyhow will put an `Error:` prefix on the message.
+            // <aybe this is slightly less clean, but it looks better for the user
```

Comment:
```suggestion
            // Maybe this is slightly less clean, but it looks better for the user
```

Reply:
Even if we `return Ok(());` ?

Reply:
I guess we need to debate if the help message is an OK or Error.
It can wait til Monday :)

Reply:
My main concern with using process::exit is when we are writing tests. I recently had this kind of situation (for plugin shutdown) where we basically call process::exit when something goes wrong and in the test we just have no clue of what's going on. But yes I think it's less problematic in this case as it's directly inside the main function

Reply:
I guess we could catch the error in the main function and translate it?

Reply:
Moved